phpstan.neon

```diff
@@ -0,0 +1,19 @@
+parameters:
+  customRulesetUsed: true
+  reportUnmatchedIgnoredErrors: false
+  # Ignore phpstan-drupal extension's rules.
+  ignoreErrors:
+    - '#\Drupal calls should be avoided in classes, use dependency injection instead#'
+    - '#Plugin definitions cannot be altered.#'
+    - '#Missing cache backend declaration for performance.#'
+    - '#Plugin manager has cache backend specified but does not declare cache tags.#'
+    #TODO: Remove when https://www.drupal.org/project/drupal/issues/2367933 is fixed.
+    - '#Call to deprecated function entity_get_display().#'
+    - '#Call to deprecated function entity_get_form_display().#'
+    #TODO: Remove when https://www.drupal.org/project/drupal/issues/2869573 is fixed.
```

Comment:
This can be fixed now - that issue will only result in a properly deprecation. See fixes on the issue for how to fix it here.